src/app/analyze/stack/container-toggler/container-toggler.component.ts

```diff
@@ -0,0 +1,25 @@
+import { Component, ViewEncapsulation, Output, EventEmitter } from '@angular/core';
+
+@Component({
+  selector: 'container-toggler',
+  encapsulation: ViewEncapsulation.None,
+  templateUrl: './container-toggler.html',
+  styleUrls: ['container-toggler.scss']
+})
+
+export class ContainerTogglerComponent {
```

Comment:
This component appears generic enough that you should move it to 'shared-component'.